x-pack/plugins/ml/common/types/ml_url_generator.ts

```diff
@@ -93,7 +93,10 @@ export interface ExplorerAppState {
   mlExplorerSwimlane: {
     selectedType?: 'overall' | 'viewBy';
     selectedLanes?: string[];
-    selectedTimes?: [number, number];
+    /**
+     * It's possible to have only "from" time boundaries, e.g. in the Watcher URL
+     */
+    selectedTimes?: [number, number] | number;
```

Comment:
The template in the watch uses `selectedTime` and `selectedLane` rather than `selectedTimes` and `selectedLanes`, and if I use an URL to test of the form

```
http://localhost:5601/iva/s/sales/app/ml#/explorer/?_g=(ml:(jobIds:!('farequote_sales')),time:(from:'2019-02-07T00:00:00.000Z',to:'2019-02-11T23:59:54.000Z'))&_a=(filters:!(),mlAnomaliesTable:(intervalValue:auto,thresholdValue:0),mlExplorerSwimlane:(selectedLane:Overall,selectedTime:1549728000,selectedType:overall),query:(query_string:(analyze_wildcard:!t,query:'*')))
``` 

the link fails to work.

Reply:
good spot, we have these URL variables renamed since `7.6`. Fixed in https://github.com/elastic/kibana/pull/89125/commits/daaab0dcffcfc6d7af588d06772e23cab17d17e6

Reply:
Will old URLs without using the plural fail to load? If yes would it be worth adding a fallback similar to some redirects we have in place to support legacy URL formats?

Reply:
Added support for legacy query params in https://github.com/elastic/kibana/pull/89125/commits/43d69cb19d72603c3c0b6abd6c4a3a52d6da0e15, cc @walterra @peteharverson